packages/app/src/renderer/store/reducers.ts

```diff
@@ -18,6 +20,7 @@ export const reducers = {
   restoreDevice: restoreDeviceReducer,
   crashDump: crashDumpReducer,
   messages: messagesReducer,
+  globalModalsManager: globalModalsManagerReducer,
```

Comment:
Let's use simpler names, like `modals` or even `modalsManager`

Reply:
`modals ` sound great. I think that is a great idea to use a simpler name until the risk of conflict 👌

Reply:
however, I kept the `manager` suffix 😅  https://github.com/mudita/mudita-center/pull/558/commits/7a3a6bd773a4d6d3e0e8f013f401a9e752e38ecb  . I thought the `modals` sounded too general 🤷‍♂️  Thanks for the inspiration 🙇